libraries/aws/provisioning/test/system/aws_iot_tests_provisioning_system.c

```diff
@@ -108,19 +109,18 @@ static const char * _pTestMqttClientId = "SystemTestClientID";
 static const AwsIotProvisioningRequestParameterEntry_t _pTestParameters[] =
     AWS_IOT_TEST_PROVISIONING_TEMPLATE_PARAMETERS;
 
-
 /**
- * @brief Type for the context parameter for the #AwsIotProvisioning_KeysAndCertificateCallbackInfo_t callback.
- * It will be used for storing the received Certificate ID and the ownership token data received from the server through
- * the callback, so that can be used for provisioning the demo application.
+ * @brief Type for the context parameter for the certificate-creation API callbacks.
+ * It will be used for storing the received Certificate ID and the ownership token
```

Comment:
"It will be used for storing the received Certificate ID and the ownership token data received from the server.  These can can then be used for the RegisterThing API test."

Reply:
Updated